src/illuminatio/illuminatio_runner.py

```diff
@@ -269,10 +269,17 @@ def store_results_to_cfg_map(results, namespace, name, runtimes=None):
     cfg_map = init_test_output_config_map(namespace, name, data=yaml.dump(results))
     if runtimes:
         cfg_map.data["runtimes"] = yaml.dump(runtimes)
-    config_map_in_cluster = api.list_namespaced_config_map(namespace, field_selector="metadata.name=" + name).items
-    if config_map_in_cluster:
+    try:
         api_response = api.patch_namespaced_config_map(name, namespace, cfg_map)
         logger.info(api_response)
-    else:
-        api_response = api.create_namespaced_config_map(namespace, cfg_map)
-        logger.info(api_response)
+    except k8s.client.rest.ApiException as e:
+        json_body = json.loads(e.body)
+        logger.debug("ApiException Body:\n%s\n" % json_body)
+        if json_body.get("code") == 404:
+            logger.info("Creating new ConfigMap")
+            api_response = api.create_namespaced_config_map(namespace, cfg_map)
+            logger.debug(api_response)
+        else:
+            logger.error("Code was: " + json_body.get("code"))
+            logger.error("An error occured while checking for an existing ConfigMap")
+            raise e
```

Comment:
Any Reason why you raise the error again ? IMHO `exit(1)` would be a better solution than raising the error again which results in a crash. We should add something like a retry if the result can't be stored. For the current state of illuminatio I would prefer a `TODO add retry logic e.g. with https://pypi.org/project/retry/` and use `exit(1)` instead of the raise.